app/streams/publishing_api_consumer.rb

```diff
@@ -11,24 +11,56 @@ def process(message)
 private
 
   def process_message(message)
-    content_id = message.payload['content_id']
-    base_path = message.payload['base_path']
+    content_id = message.payload.fetch('content_id')
+    base_path = message.payload.fetch('base_path')
+    payload_version = message.payload.fetch('payload_version')
     locale = message.payload['locale'] || 'en'
 
     item = Dimensions::Item.by_natural_key(content_id: content_id, locale: locale)
+
     if item
-      handle_existing(item, base_path, message.delivery_info.routing_key)
+      existing_payload_version = item.publishing_api_payload_version || 0
```

Comment:
Not sure about this line. What is the intention here? to prevent running a migration to update all the payload versions to 0? Can we have versions with not `api_payload_version`? 

If the event always has a version, I would rather making it explicit by:
1. Running a migration to ensure that existing payloads a 0 version number.
2. Preventing nil version at the database/model level

If the event could have a nil version, this would entail that we would never be updating those events.

Reply:
Yeah this was to avoid a migration. I can do that.